wazuh/wazuh-kibana-app
# Remove trailing `-` character for OS value in the list of agents

### Description
Team,
this PR removes the trailing `-` character for OS version is empty value in the list of agents
 
### Issues Resolved
Closes https://github.com/wazuh/wazuh-kibana-app/issues/4809

### Evidence
![image](https://user-images.githubusercontent.com/9343732/200576219-a8d6964b-9b54-41af-90ca-8489f37c4919.png)


### Test

- Register 2 or more agents in the manager
- Create at least 2 groups
- Add 1 on the agents to both groups and remove it from the default group
- Register another agents from _Wazuh Dev tools_ using this request
```
POST /agents
{
  "name": "NewAgent"
}
```

- Go to the Agents table
- Verify empty fields contain a `-` value
- Verify the OS column has a `-` only if it's empty or as a separator between the OS name and version
- Verify that if the OS has only the name, but has no version then it doesn't include a `-` separator

### Check List
